### ISSUE TITLE
Unable to install "processing-java"

### ISSUE THREAD

**Issue by neilpanchal** [processing/processing#3224]
On version 3.0a5, I am able to install  `processing-java` command under Toos > Install "processing-java".

On version 3.0a7, I get the following error:
`Error: Could not find or load main class processing.mode.java.Commander`

I've tried searching for this problem and looks like its an issue with the ".class" extension during build.

Hope this helps,
Neil

**Comment by benfry** [comment]
Thanks for the report, sorry we didn't run into this during development, but I'll have a fix for the next release (and a workaround sooner).

**Comment by benfry** [comment]
Fixed for 3.0a8. 

In the meantime, you can manually fix your installation by first using the tool, then editing `/usr/bin/processing-java` with a text editor. Replace everything that starts with `-cp` with the following:

```
-cp "ant-launcher.jar:ant.jar:core.jar:jna.jar:pde.jar:core/library/core.jar:core/library/lwjgl.jar:core/library/lwjgl_util.jar:modes/java/mode/antlr.jar:modes/java/mode/classpath-explorer-1.0.jar:modes/java/mode/com.ibm.icu.jar:modes/java/mode/JavaMode.jar:modes/java/mode/jdi.jar:modes/java/mode/jdimodel.jar:modes/java/mode/jdtCompilerAdapter.jar:modes/java/mode/jsoup-1.7.1.jar:modes/java/mode/org.eclipse.core.contenttype.jar:modes/java/mode/org.eclipse.core.jobs.jar:modes/java/mode/org.eclipse.core.resources.jar:modes/java/mode/org.eclipse.core.runtime.jar:modes/java/mode/org.eclipse.equinox.common.jar:modes/java/mode/org.eclipse.equinox.preferences.jar:modes/java/mode/org.eclipse.jdt.core.jar:modes/java/mode/org.eclipse.osgi.jar:modes/java/mode/org.eclipse.text.jar:modes/java/mode/org.netbeans.swing.outline.jar" processing.mode.java.Commander "$@"
```
